src/pages/civic/index.vue

```diff
@@ -147,8 +142,12 @@
                 .flex.justify-center.mb-24(
                   v-if="isExperimenting"
                 )
-                  LikeButtonAnimation(:size="180")
+                  LikeButtonAnimation(
+                    :size="180"
+                    :avatar="referrer.avatar"
+                  )
                   LcChopCivicLiker(
+                    v-if="!referrer.avatar"
```

Comment:
will this explode if referrer is undefined?

Reply:
I have guarded in L143 & L243